src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -67,7 +67,7 @@ public FeedbackResponseCommentAttributes createFeedbackResponseComment(
                 }
                 frComment.setId(existingComment.getId());
                 
-                return frcDb.updateFeedbackResponseComment(frComment);            
+                return frcDb.updateFeedbackResponseComment(frComment, true);            
```

Comment:
the boolean here doesn't provide much information.
may need to extract the method